lib/processors/bundlers/moduleBundler.js

```diff
@@ -35,8 +35,11 @@ const log = getLogger("builder:processors:bundlers:moduleBundler");
  *	<li>
  *		<code>require</code>: A 'require' section is transformed into a sequence of jQuery.sap.require calls. The
  *		list will be resolved like an include pattern list in any of the other sections and for each of the resolved
- *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.requireSync' is
+ *		modules, a jQuery.sap.require will be created. In case the ui5loader is available, 'sap.ui.require' is
  *		used instead.
+ *		With specVersion: 4.0 this module comes with an `async` flag. When set to false, the modules
```

Comment:
Is the moduleBundler aware of a specVersion? IMO it just has an API interface with certain defaults, but the processor is completely independent of any project configuration.

Reply:
Done